File: data-access-provider-archetype/src/main/resources/archetype-resources/src/main/java/__pluginName__/__pluginName__DataAccessProviderConfig.java
package ${package}.${pluginName};

import se.curity.identityserver.sdk.config.Configuration;
import se.curity.identityserver.sdk.config.annotation.Description;

public interface ${pluginName}DataAccessProviderConfig extends Configuration
{
    @Description("A parameter used for demo purposes")
    String getDemoConnectionUri();
}

File: kotlin-data-access-provider-archetype/src/main/resources/archetype-resources/src/main/kotlin/__pluginName__/__pluginName__DataAccessProviderConfig.kt
package ${package}.${pluginName}

import se.curity.identityserver.sdk.config.Configuration
import se.curity.identityserver.sdk.config.annotation.Description

interface ${pluginName}DataAccessProviderConfig: Configuration
{
    @Description("A parameter used for demo purposes")
    fun getDemoConnectionUri(): String
}
